src/styles/components/header-bar/styles.less

```diff
@@ -65,9 +57,81 @@
         .triangle-size-height('down', @header-dropdown-caret-height);
       }
     }
-  }
 
-  .header-bar-dropdown-menu.down {
-    margin-top: 10px;
+    .header-bar-right-align-wrapper {
+      display: inline-flex;
+      justify-content: flex-end;
+      overflow: hidden;
+      width: 100%;
+
+      .header-bar-dropdown-trigger {
+        color: @purple-lighten-4;
+        cursor: pointer;
+        display: inline-flex;
+        padding-left: 15px;
+
+        &:hover {
+          color: @white;
+
+          &:after {
+            .header-bar-caret(@white);
+          }
+        }
+
+        &:after {
+          .header-bar-caret(@purple-lighten-4);
+        }
+
+        .header-bar-dropdown-content {
+
+          & when (@layout-screen-small-enabled) {
+
+            @media (max-width: @layout-screen-iteration-10) {
```

Comment:
Uff… I'm not that familiar with css nowadays, but isnt there a better solution to this? 😢